app/cdap/components/WrangleHome/index.tsx

```diff
@@ -51,13 +59,25 @@ export default function() {
           </Box>
         </Box>
         <WrangleCard />
-        <Box className={classes.headerTitle}>
-          <WrangleHomeTitle title={T.translate('features.HomePage.labels.workspaces.title')} />
-          <Box className={classes.viewMore}>
-            {T.translate('features.HomePage.labels.common.viewAll')}
+        {showExplorations ? (
```

Comment:
can we do showExplorations && JSX.Element ??
will this add any errors or unwanted 0s
plz check